app/helpers/and.js

```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
+import { andHelper } from 'ember-truth-helpers/helpers/and';
+
+export default Ember.Helper.helper(andHelper);
```

Comment:
Note that we don't check for `Ember.Helper` here because this files should be auto-discovered only in Ember 1.13, which has `Ember.Helper` API.